lib/object_store/app/validators/object_store/content_block_validator.rb

```diff
@@ -0,0 +1,27 @@
+require "json-schema"
+
+# TODO: Rename "validator" to something else
+class ObjectStore::ContentBlockValidator
+  EMAIL_ADDRESS_SCHEMA = Rails.root.join("lib/object_store/config/object_store/schemas/email_address.json")
```

Comment:
we now have the `/schemas` endpoint on the Publishing API so maybe this would be a call to that/that would handle validation?

Reply:
Yeah, I think we should be able to use the endpoints we setup before now

Reply:
However, we'll probably need to add a schema first (though we could/should stub out in test) - I'll look into that

Reply:
I've had a look at the schemas on Publishing API and I'm not sure we can use without changing the current configuration quite dramatically - there's a generation process that adds / assumes a lot about the fields and schema  https://github.com/alphagov/publishing-api/blob/9dc6d2a7c36809069748cb5457465bebf983c558/docs/content_schemas/adding-a-new-schema.md